app/src/main/java/com/lagradost/cloudstream3/utils/JsHunter.kt

```diff
@@ -0,0 +1,90 @@
+package com.lagradost.cloudstream3.utils
+
+import com.lagradost.cloudstream3.mvvm.logError
+import java.util.regex.Matcher
+import java.util.regex.Pattern
+import kotlin.math.pow
+
+//author: https://github.com/daarkdemon
+class JsHunter(private val hunterJS: String) {
+
+    /**
+     * Detects whether the javascript is H.U.N.T.E.R coded.
+     *
+     * @return true if it's H.U.N.T.E.R coded.
+     */
+    fun detect(): Boolean {
+        val js = hunterJS
+        val p = Pattern.compile("eval\\(function\\(h,u,n,t,e,r\\)")
+        val searchResults = p.matcher(js)
+        return searchResults.find()
+    }
+
+    /**
+     * Unpack the javascript
+     *
+     * @return the javascript unhunt or null.
```

Comment:
Or null?
Right now its non-nullable, though I'd personally prefer if it was nullable

Reply:
made it nullable